ts/torch_handler/base_handler.py

```diff
@@ -67,6 +66,9 @@ def initialize(self, context):
             self.model = self._load_pickled_model(model_dir, model_file, model_pt_path)
         else:
             logger.debug("Loading torchscript model")
+            if not os.path.isfile(model_pt_path):
```

Comment:
Looks like presence / absence of modelFile arg in the manifest determines if this is an eager mode / torchscript model. Given usage of this in AWS Inf tolkit 

```
    model_archiver_cmd = [
        "torch-model-archiver",
        "--model-name",
        DEFAULT_TS_MODEL_NAME,
        "--handler",
        handler_service,
        "--serialized-file",
        os.path.join(environment.model_dir, DEFAULT_TS_MODEL_SERIALIZED_FILE),
        "--export-path",
        DEFAULT_TS_MODEL_DIRECTORY,
        "--extra-files",
        os.path.join(environment.model_dir, DEFAULT_TS_CODE_DIR, environment.Environment().module_name + ".py"),
        "--version",
        "1",
    ]

```

https://github.com/aws/sagemaker-pytorch-inference-toolkit/blob/6936c08581e26ff3bac26824b1e4946ec68ffc85/src/sagemaker_pytorch_serving_container/torchserve.py#L110

Does AWS Inf toolkit even support Torchscript models ?

Reply:
toolkit missed option "--model-file " for eager mode. It should be fixed in toolkit.

Reply:
If customers have been using TS archiver without `--model-file` args, we should make sure this doesn't cause any backward compatibility issues

Reply:
On TS side, the only change for option "--serialized-file" is that it is optional in eager mode. So it is backward compatible.